examples/benchmark/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2016, Dennis Sitelew
+cmake_minimum_required(VERSION 3.0)
+
+if(NOT CMAKE_BUILD_TYPE)
+  message(WARNING "You shouldn't benchmark a debug build")
```

Comment:
I can recommend you a good video on benchmarking: ttps://www.youtube.com/watch?v=nXaxk27zwlk

In short: it doesn't make any sense you benchmarking nothing since it isn't the code you will run on your production system.

As for detecting debug build it isn't required, just a hit for developers who will look at examples and try to implement their own benchmarks. Maybe I should add more rationale behind this message or delete it entirely.